tests/model/test_model.py

```diff
@@ -1597,3 +1604,47 @@ def test_is_muted_topic(self, topic, is_muted, stream_dict, model):
         ]
         assert model.is_muted_topic(stream_id=topic[0],
                                     topic=topic[1]) == is_muted
+
+    @pytest.mark.parametrize('response', [
+        {
+            'emoji': {
+                '100': {
+                    'author_id': 5,
```

Comment:
These appear to be user ids. If so, we could correlate these to ones we use in other fixtures?

We may want to take into account the variation between 2.1 and 3+ here due to https://www.zulipchat.com/api/get-custom-emoji

Reply:
We're not using this right now, it would be simpler if I just remove the attribute?